src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -205,6 +205,15 @@ public async Task GetContentAsync_ErrorStatusCode_ExpectedExceptionThrown(bool w
             }
         }
 
+        [Fact]
+        [OuterLoop("Slow - Negative connection test")]
+        public async Task GetContentAsync_WhenCanNotConnect_ExceptionContainsHostInfo()
```

Comment:
This should likely belong to a `SocketsHttpHandler` test, but I'm new to HTTP codebase, and couldn't figure out where. Any suggestion?

Reply:
I think this is a good enough place for this test.